composer.json

```diff
@@ -35,14 +35,20 @@
         "league/container": "^2.4",
         "aura/sqlquery": "3.*-dev",
         "twig/twig": "^2.0",
-        "slim/slim": "^3.0"
+        "slim/slim": "^3.0",
+        "phpmailer/phpmailer": "5.2.*"
     },
     "require-dev": {
         "twbs/bootstrap": "^4.1"
     },
     "replace": {
         "pimple/pimple": "*"
     },
+    "config": {
+        "platform": {
+            "php": "7.0"
```

Comment:
The docs say this configuration "Lets you fake platform packages" https://getcomposer.org/doc/06-config.md#platform Can you let me know why we might need to do this (since we're using real php, why does it need to be faked?) Thanks!

Reply:
Because I am using PHP 7.2, if my machine is resolving the new composer.lock, they will not be compatible with 7.0. This is to ensure that the versions in lock is compatible with the lower bound of target compatibility.

Reply:
This also works for developers with similar situation in the future.

Reply:
Okay, that makes sense. Thanks!